src/style/book.css

```diff
@@ -0,0 +1,90 @@
+.BookClass {
```

Comment:
Good job with the general styling, however, some of the elements do not match the Zeplin design closely enough.

Look:

![image](https://user-images.githubusercontent.com/57421823/161324894-1613baad-3cce-42be-8836-4dd050c0d46e.png)

Therefore:

- [ ] Please, kindly correct the margins/paddings for the elements of each book, so their margins and padding are closer to the Zeplin design.
- [ ] The links below each book in your project should feature 1 vertical line after each element, not before.
- [ ] Kindly correct the aforementioned vertical lines, so the color matches the Zeplin.
- [ ] After the book collection, the big horizontal line does not match the Zeplin.  Please correct its color (should feature some transparency.
- [ ] Please, correct the borders featured on the inputs "Book Title", "Author" and "Category", so their color matches.
- [ ] Please, kindly remove the border on the "Add book" button, and make it a little bigger, like in the Zeplin.
- [ ] Please, make your borders a little rounded on the corners, you can use the CSS property "border-radius".
- [ ] Kindly make the word "completed" Capitalized on the first letter.

If you implement these changes, your project will look more like the Zeplin 💪